sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/MesosResourcePool.java

```diff
@@ -115,6 +119,10 @@ public Offer getOffer() {
         return Optional.empty();
     }
 
+    public void retainOnly(Offer offer) {
```

Comment:
Javadoc comment please.  Not immediately obvious to me why the word `retain` is used.